pkg/project/app_classic/Taskfile.yml

```diff
@@ -149,12 +149,28 @@ tasks:
 
   aem:build:
     desc: build AEM application using Maven
+    sources:
```

Comment:
there might be a problem with exclusions - see 

https://github.com/go-task/task/issues/225
https://github.com/go-task/task/pull/309/files